src/v/config/configuration.cc

```diff
@@ -2813,6 +2813,14 @@ configuration::configuration()
       "Per-shard capacity of the cache for validating schema IDs.",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       128)
+  , pp_sr_smp_max_non_local_requests(
+      *this,
+      "pp_sr_smp_max_non_local_requests",
+      "Maximum number of x-core requests pending in Panda Proxy and Schema "
+      "Registry seastar::smp group.  (for more details look at "
+      "`seastar::smp_service_group` documentation)",
+      {.needs_restart = needs_restart::yes, .visibility = visibility::user},
+      std::nullopt)
```

Comment:
I don't believe we backport things if there's a new cluster config (right?).  If we do want this backported, I can drop the commits that add this and submit the change in a separate PR.